tests/RobotFramework/tests/cumulocity/availability/c8y_required_availability.robot

```diff
@@ -0,0 +1,66 @@
+*** Settings ***
+Resource    ../../../resources/common.resource
+Library    Cumulocity
+Library    ThinEdgeIO
+
+Test Tags    theme:c8y
+Test Setup    Test Setup
+Test Teardown    Get Logs
+
+*** Test Cases ***
+
+c8y_RequiredAvailability is set by default
+    Execute Command    ./bootstrap.sh
```

Comment:
```suggestion
    Setup
```
Why not just that I wonder.

Reply:
`Setup` does a lot more than just bootstrapping a device. When using the docker adapter it will spawn a new container. The `Test Setup` is already creating the container (but with bootstrapping disabled), as the tests look to require fine-grain control over the initial connection.